=13.3.3

```diff
@@ -0,0 +1,4 @@
+Requirement already satisfied: rich in /opt/anaconda3/envs/myenv/lib/python3.11/site-packages (13.3.2)
+Requirement already satisfied: markdown-it-py<3.0.0,>=2.2.0 in /opt/anaconda3/envs/myenv/lib/python3.11/site-packages (from rich) (2.2.0)
+Requirement already satisfied: pygments<3.0.0,>=2.13.0 in /opt/anaconda3/envs/myenv/lib/python3.11/site-packages (from rich) (2.18.0)
+Requirement already satisfied: mdurl~=0.1 in /opt/anaconda3/envs/myenv/lib/python3.11/site-packages (from markdown-it-py<3.0.0,>=2.2.0->rich) (0.1.2)
```

Comment:
Feel free to ignore these auto-added checks when merging! They seem to have been added automatically when I pushed, though I’m not sure why or where they came from 😅. Rest assured, they should not affect the behavior of the code.